vector-im/element-android
# Deferred DM - Add a loading wheel while creating the DM

## Type of change


## Content

While creating the DM (ie. after sending the first message), a loading wheel is now displayed until switching to the created room.
In case of failure, the loading wheel is hidden to unblock the UI.

## Motivation and context

close #6970 

## Screenshots / GIFs


## Tests

- Create a DM on first message
- Verify that a loading wheel is shown during the process creation

## Tested devices

- OS version(s):

## Checklist
